src/index.js

```diff
@@ -13,6 +13,20 @@ const app = express();
 
 dotenv.config();
 
+app.use(cors());
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
```

Comment:
great job @urbanishimwe  but basically you have used two methods:
- you have used CORS package
- you have also created cors middleware your self.
so basically choose among the two using a package or implementing it  your self

Reply:
Please stick to one method, preferably the first.

Reply:
okay, right away!